lbz/configuration.py

```diff
@@ -1,13 +1,30 @@
+import json
 from abc import ABCMeta, abstractmethod
 from os import getenv
-from typing import Any, Callable, Generic, Optional, TypeVar
+from typing import Any, Callable, Generic, List, Optional, TypeVar, cast
 
 from lbz.aws_ssm import SSM
 from lbz.exceptions import ConfigValueParsingFailed, MissingConfigValue
 
 T = TypeVar("T")
 
 
+class ConfigParser:
+    @staticmethod
+    def split_by_comma(cfg: str) -> List[str]:
+        return cfg.split(",")
+
+    @staticmethod
+    def env_bool(cfg: str) -> bool:
```

Comment:
Why `env`? Exactly the same string could be stored in SSM 😉

Reply:
Also, to keep naming consistent, it should be rather named as a command e.g. `cast_to_bool` or something similar.